x-pack/plugins/ml/public/components/job_select_list/job_select_service.js

```diff
@@ -159,7 +158,7 @@ export function JobSelectServiceProvider($rootScope, globalState, i18n) {
           const total = (wholeGroups.length - 1) + groupLessJobs;
           txt = i18n('xpack.ml.jobSelect.wholeGroupDescription', {
             defaultMessage: `{wholeGroup} (with {count, plural, zero {# job} one {# job} other {# jobs}}) and
```

Comment:
I just noticed that the work `group` is missing here:
`{wholeGroup} group (with {count.....`

![image](https://user-images.githubusercontent.com/22172091/53794353-b4235680-3f27-11e9-99c8-653822f0c571.png)

Reply:
Ignore this comment. I made a mistake with the name of my group being `1`.
Nothing needs to change